src/services/database/schemata.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+const schemaVZero = require('./schema/v0');
```

Comment:
Can we use import here instead of require? It gives more consistency with other project parts.

Reply:
done